dandalandone/POC-EyCommerce
# [API Implementation] Get all products

## Ticket/s


## Changes
- [API] Added Get all products
- Add sample API implementation

## Screenshot
![image](https://github.com/dandalandone/POC-EyCommerce/assets/130529098/7d969626-c747-414a-8727-08f63a1caf5a)

## How to test?
- Run everything
- Go to http://localhost:5173/products then check if everything works.
